src/AspNetCore.Lambda/HttpHandlers.fs

```diff
@@ -140,10 +143,22 @@ let clearResponse =
         ctx.HttpContext.Response.Clear()
         async.Return (Some ctx)
 
+let private getRoute (ctx:HttpContext) =
+    if ctx.Items.ContainsKey RouteKey
+    then
+        let x = ctx.Items.Item RouteKey |> string 
+        if String.IsNullOrEmpty x then None else Some x 
+    else None
+
+let private getRalativePath (ctx:HttpContext) =
```

Comment:
Yeah I had a quick look as well.. I think the approach you are taking looks good.

Re naming, I'd prefer something more like this perhaps:

```
let private strOption (str : string) =
    if String.IsNullOrEmpty str then None else Some str 

let private getSubPath (ctx : HttpContext) =
    if ctx.Items.ContainsKey RouteKey
    then ctx.Items.Item RouteKey |> string |> strOption 
    else None

let private getPath (ctx : HttpContext) =
    match getSubPath ctx with
    | Some route -> ctx.Request.Path.ToString().[route.Length..]
    | None       -> ctx.Request.Path.ToString()
```